include/plog/Util.h

```diff
@@ -189,18 +189,25 @@ namespace plog
 #endif
         }
 
-#ifdef _WIN32
+#if defined(_MSC_VER) && _MSC_VER <= 1600
+#   define va_copy(d,s) ((d)=(s)) // there is no va_copy on Visual Studio 2010
```

Comment:
We can't do this as a macro as macros are global and not confined to namespaces. Thus it can have side-effects in interaction with other code. It's better to have it as a function.

Reply:
I decided to move these macros into the vasprintf function.